### ISSUE TITLE
Provide guidance when the final project packaged file is not found

### ISSUE THREAD

**Issue by sunix** [eclipse/jkube#1328]
### Description
When running `ocBuild` or `k8sBuild` without having built the project, the following message is display
```
* What went wrong:
Execution failed for task ':ocBuild'.
> Final output artifact file was not detected
```
I did not understand right away.
JKube should guide the user with an message like

```
* What went wrong:
Execution failed for task ':ocBuild'.
> The final project packaged file was not found. The project may have not been built. HINT: try to execute the `build` task prior to `ocBuild` task.
```

### Info

* Eclipse JKube version : 1.7.0

* Sample Reproducer Project : tested with quickstarts/gradle/webapp

**Comment by Urjaswi-2655** [comment]
@sunix @rohanKanojia @manusa  I want to work on this issue due to its label(good first Issue)
I haven't run  ocBuild or k8sBuild without having built the project to check the message displayed above 
But I want to confirm that my work would be to change the displayed message.
Question: Where should I change the message and How ?
And can you help me  to setup JKube , So that I can understand better about displayed errors in issues .
Would this will help to do my setup (https://www.eclipse.org/jkube/contributing)  && (https://github.com/eclipse/jkube/tree/master/quickstarts/maven/hello-world/)
Thanks
And should I start learning about Maven and then testing (JUnit 5 and Mockito ) to be able to contribute to GSoC.
And to know basic of  Kubernetes, Docker and OpenShift would be helpful too.

**Comment by Prathamkrishna** [comment]
hey @rohanKanojia I'd like to work on this

**Comment by manusa** [comment]
Fixed by #1408
